content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -1,99 +1,190 @@
-======================
-Request a down payment
-======================
+=============
+Down payments
+=============
 
 A down payment is a partial payment made by the buyer when a sales contract is concluded. This
-implies both parties' full commitment (seller and buyer) to honor the contract. With a down payment,
-the buyers show their will to acquire the product and agree to pay the rest later, while the sellers
-are obliged to provide the goods by accepting it.
+implies both parties' (seller and buyer) full commitment to honor the contract.
 
-Sometimes a down payment is required for expensive orders or projects. That way, you can protect
-yourself and make sure that your customer is reliable.
+With a down payment, the buyer shows their will to acquire the product, and provides a portion of
+the total payment, agreeing to pay the remaining amount at a later date. In turn, the seller is
+obliged to provide the goods to the buyer, by accepting the down payment, trusting that the
+remaining amount will eventually be paid.
```

Comment:
Reads a little strange because of the verb tense and possible missing conjunctions.
Here's a suggested rewrite which also cuts down a line:
```suggestion
With a down payment, the buyer pays a portion of the total amount owed while agreeing to pay the
remaining amount at a later date. In turn, the seller provides goods or services to the buyer after
accepting the down payment, trusting that the remaining amount will be paid later on.
```